ios/ExpensifyCash-Bridging-Header.h

```diff
@@ -0,0 +1,4 @@
+//
+//  Use this file to import your target's public headers that you would like to expose to Swift.
```

Comment:
Is this file required it seems blank?

Reply:
+1

Reply:
What was the resolution here? I see these files were removed but there's still a `BridgingFile.swift` committed. Is that file necessary?